data/translations/en/state-data.json

```diff
@@ -749,27 +834,31 @@
     "more_info_link": "https://www.scvotes.gov/south-carolina-voter-registration-information",
     "more_info_link_translated": "true",
     "more_info_link_language_selection": "",
-    "ip_deadline": "30 days before Election Day",
-    "online_deadline": "30 days before Election Day",
-    "mail_received_deadline": "",
-    "mail_postmarked_deadline": "Must be postmarked 30 days before Election Day",
+    "online_deadline": "Sunday, October 09, 2022",
+    "ip_deadline": "Sunday, October 09, 2022",
+    "mail_received_deadline": "Must be received by Tuesday, October 11, 2022",
+    "mail_postmarked_deadline": "Must be postmarked by Tuesday, October 11, 2022",
     "confirm_registration_link": "https://info.scvotes.sc.gov/eng/voterinquiry/VoterInformationRequest.aspx?PageMode=VoterInfo",
     "confirm_registration_link_translated": "true",
-    "confirm_registration_link_language_selection": ""
+    "confirm_registration_link_language_selection": "",
+    "default_ip_deadline": "30 days before Election Day",
+    "default_online_deadline": "30 days before Election Day",
+    "default_mail_postmarked_deadline": "Must be postmarked 30 days before Election Day"
   },
   "sd": {
     "hp_link": "https://sdsos.gov/elections-voting/",
     "hp_link_translated": "true",
     "hp_link_language_selection": "",
     "more_info_link": "https://sdsos.gov/elections-voting/voting/register-to-vote/default.aspx",
     "more_info_link_translated": "true",
-    "more_info_link_language_selection": "",
-    "ip_deadline": "15 days before Election Day",
-    "mail_received_deadline": "Must be received 15 days before Election Day",
-    "mail_postmarked_deadline": "",
+    "more_info_link_language_selection": "Monday, October 24, 2022",
```

Comment:
This date was applied to the wrong property.

Reply:
updated